src/api/sso/src/index.js

```diff
@@ -8,6 +8,7 @@ const routes = require('./routes');
 require('./authentication');
 
 const service = new Satellite({
+  cors: { origin: 'https://telescope.cdot.systems', credentials: true },
```

Comment:
The trouble we'll have here is that the origin is different depending on how you run the app: dev, staging, production.

We store this value in an environment variable named `WEB_URL`.  It's different on every deployment.  So here you should use `process.env.WEB_URL`.

However, we don't seem to expose this to our SSO container, see https://github.com/Seneca-CDOT/telescope/blob/master/docker/docker-compose.yml#L74-L96.  You need to add it there.

And in our local dev env file https://github.com/Seneca-CDOT/telescope/blob/master/src/api/sso/env.local, which should mimic what we have in https://github.com/Seneca-CDOT/telescope/blob/master/config/env.development#L31.

There might be other things we need to make this work, but that's a start.

Reply:
I will try this soon, is it ok if this takes a day or two? I am backed up a bit on other school work.

Reply:
Yes, leave this for a while, it's not important in the next week or so.